nucleus/common/common-util/src/main/java/com/sun/enterprise/security/store/AsadminSecurityUtil.java

```diff
@@ -40,10 +40,13 @@
 
```

Comment:
Needs Portions Copyright

Reply:
If I look at the "Files Changed" at the beginning of the source file I can see the copyright comments, please advice if "Portions Copyright" is something else I should add and where I can find it. Sorry just first time I am using github to pull something :)

Reply:
About indentation, I fixed and committed it on my fork of the site, you should be able to see the new version. Thanks

Reply:
No problem, thanks for fixing the indentation :)
The portions copyright is an addition we need to add to any modified file - it looks like this:
`// Portions Copyright [2018] [Payara Foundation and/or its affiliates]`

You can see an example here: https://github.com/payara/Payara/blob/master/appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/runtime/ConcurrentRuntime.java#L40

Reply:
Thanks, I have just added the Portions Copyright. About the CLA, I do not know what is it :(  let me know if there is something I should do.

Reply:
Can you update the copyright to just the current year please? :)

Regarding the CLA, we need you to sign [this CLA ](https://github.com/payara/Payara/blob/master/PayaraCLA.pdf) and send it to cla@payara.org (or you can post it to the address specified in the CLA if you prefer :P)

Sorry for all this, it's all a bit of a faff but that's how copyright licensing works unfortunately.

Reply:
I did the update, about CLA I asked to my company and I still waiting for the confirmation

Reply:
I have just sent the CLA